### ISSUE TITLE
Text Motion Render result with wrong axis

### ISSUE THREAD

**Issue by Zessay** [GuyTevet/MotionCLIP#11]
I test the `text_motion` script according to the readme, but the body in the render result is opposite. I don't modify any config and have no idea to solve this.

<img width="922" alt="image" src="https://user-images.githubusercontent.com/39905704/189262646-a69afb79-79e7-43c7-a475-a1d22fe9ebeb.png">

**Comment by GuyTevet** [comment]
Hi @Zessay ! What dataset did you use? please share the cmd you run to produce this result.

**Comment by Zessay** [comment]
> Hi @Zessay ! What dataset did you use? please share the cmd you run to produce this result.

Hi, I use the pretrained model in the readme and just test with the given command line 

```shell
python -m src.visualize.text2motion ./exps/paper-model/checkpoint_0100.pth.tar --input_file assets/paper_texts.txt
```

**Comment by larsh0103** [comment]
Yeah, I also got this bug. For some reason the frames are flipped, I don't know the source of this issue. But a dirty fix is to just rotate the pixels 180 degrees. For instance adding this  `    for i in range(frames.shape[0]):
        for j in range(frames.shape[3]):
            frames[i,:,:,j] = np.rot90(frames[i,:,:,j],k=2)`

to the visualization functions before writing the frames to file. Worked for me.

**Comment by larsh0103** [comment]
I had the same issue, and I suppose its due to some miss match between data formats, for amass data the axes need to be inverted. There is a hack in anim.py that solves this issue, just uncomment line 105 and 106     # # invert axis
    motion[:, 1, :] = -motion[:, 1, :]
    motion[:, 2, :] = -motion[:, 2, :]
    # this hack is not needed for amass

**Comment by ywdong** [comment]
@larsh0103 Thx your tips! Do u know how to render the image from the text-to-emotion script?
